### PR TITLE
Add pull request templates

### PR DESCRIPTION
**Pull request by brucellino**
This PR adds two pull request templates, each in English and Italian (4 files in total).

The two kinds of PR are :

- generic PR (for most kinds of PR)
- contribution PR (for PRs including contributions of derivative works)

Signed-off-by: Bruce Becker <bruce.becker@uefa.ch>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/contributo.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+<!--
+Fornite una breve descrizione del vostro contributo.
+-->
+
+---
+
+- **Tipo di contributo**: <!-- Che tipo di contributo è? e.g. rielaborazione dati, nuova visualizazzione, prodotto scientifico, ecc. -->
+- **Valore di pubblico interesse:** <!-- nella vostra stima, in che modo questo contributo aggiunge al valore pubblico di questo repo? -->
+- **Contribuenti** <!-- elenco di persone (github username / email) che hanno partecipato, in che misura, se vanno aggiunti a CONTRIBUTORS -->
```
Comment (simogasp) [inline .github/PULL_REQUEST_TEMPLATE/contributo.md]:
```suggestion
- **Contributori** <!-- elenco di persone (github username / email) che hanno partecipato, in che misura, se vanno aggiunti a CONTRIBUTORS -->
```
Contribuenti is taxpayers.
Altra possibilita': partecipanti

File: .github/PULL_REQUEST_TEMPLATE/contribution.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+<!--
+Provide a brief description of your contribution.
+-->
+
+---
+
+- **Type of contributon**: <!-- What type of contribution would you say this is ? e.g. Analysis of the data, new visualisation, scientific product, etc. -->
+- **Valore di pubblico interesse:** <!-- In your opinion, what is the public value of this contribution? -->
+- **Contribuenti** <!-- Provide a list of people (github usernames / email addresses ) who participated to this contribution, if you would like them to be added to the CONTRIBUTORS file.-->
```
Comment (simogasp) [inline .github/PULL_REQUEST_TEMPLATE/contribution.md]:
```suggestion
- **Contributori** <!-- Provide a list of people (github usernames / email addresses ) who participated to this contribution, if you would like them to be added to the CONTRIBUTORS file.-->
```
o partecipanti

**Comment by brucellino** [comment]
Grazie @simogasp mi suonava strano!

**Comment by tehKapa** [comment]
> Does this apply both for PR and Issues?

No, this only templates for PR. Issues have similar templates but located on [.github/ISSUE_TEMPLATE/](https://github.com/pcm-dpc/COVID-19/tree/master/.github/ISSUE_TEMPLATE)

**Pull request closed without merging**